flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestCatalogTableLoader.java

```diff
@@ -58,9 +60,11 @@ public static void createWarehouse() throws IOException {
   }
 
   @AfterClass
-  public static void dropWarehouse() {
+  public static void dropWarehouse() throws IOException {
     if (warehouse != null && warehouse.exists()) {
-      warehouse.delete();
+      Path warehousePath = new Path(warehouse.getAbsolutePath());
+      FileSystem fs = warehousePath.getFileSystem(hiveConf);
+      Assert.assertTrue("Failed to delete " + warehousePath, fs.delete(warehousePath, true));
```

Comment:
Taking a fresh look at the code here, I think we might want to do this cleanup in the `TestHiveMetastore.stop()`.
We already clean up the warehouse root in `TestHiveMetastore.stop()`, we might want to clean up the data in the stop as well.

What do you think?

Reply:
Ok.. this is for other warehouses as well